source/guides/guides/network-requests.md

```diff
@@ -358,31 +314,30 @@ cy.get('#results')
 **Examples**
 
 ```javascript
-cy.server()
 // spy on POST requests to /users endpoint
-cy.route('POST', '/users').as('new-user')
+cy.http('POST', '/users').as('new-user')
 // trigger network calls by manipulating web app's user interface, then
 cy.wait('@new-user')
-  .should('have.property', 'status', 201)
+  .should('have.property', 'response.statusCode', 201)
 
-// we can grab the completed XHR object again to run more assertions
+// we can grab the completed request object again to run more assertions
 // using cy.get(<alias>)
-cy.get('@new-user') // yields the same XHR object
-  .its('requestBody') // alternative: its('request.body')
-  .should('deep.equal', {
+cy.get('@new-user') // yields the same request object
+  .its('request.body')
+  .should('deep.equal', JSON.stringify({
     id: '101',
     firstName: 'Joe',
     lastName: 'Black'
-  })
+  }))
 
 // and we can place multiple assertions in a single "should" callback
 cy.get('@new-user')
-  .should((xhr) => {
-    expect(xhr.url).to.match(/\/users$/)
-    expect(xhr.method).to.equal('POST')
+  .should(({ request, response }) => {
+    expect(request.url).to.match(/\/users$/)
+    expect(request.method).to.equal('POST')
     // it is a good practice to add assertion messages
     // as the 2nd argument to expect()
-    expect(xhr.response.headers, 'response headers').to.include({
+    expect(response.headers, 'response headers').to.include({
       'cache-control': 'no-cache',
       expires: '-1',
       'content-type': 'application/json; charset=utf-8',
```

Comment:
maybe there's something wrong with my examples? This doesn't work for me either. 

```js
  it('cy.http() - route responses to matching requests', () => {
    cy.visit('https://example.cypress.io/commands/network-requests')

    cy.http('POST', '/comments').as('postComment')
    cy.get('.network-post').click()

    cy.wait('@postComment')

    cy.get('@postComment')
    .should(({ request, response }) => {
      expect(request.url).to.match(/\/users$/)
      expect(request.method).to.equal('POST')
      // it is a good practice to add assertion messages
      // as the 2nd argument to expect()
      expect(response.headers, 'response headers').to.include({
        'cache-control': 'no-cache',
        expires: '-1',
        'content-type': 'application/json; charset=utf-8',
      })
    })
})
```

<img width="842" alt="Screen Shot 2020-11-20 at 2 33 17 PM" src="https://user-images.githubusercontent.com/1271364/99775244-5e4c7300-2b3d-11eb-927d-4a5745de76cb.png">